package.json

```diff
@@ -39,10 +40,14 @@
     "atom-space-pen-views": "^2.0.5",
     "escape-string-regexp": "^1.0.5",
     "jmp": "0.4.x",
+    "jupyter-js-services": "^0.11.4",
```

Comment:
Not really. I think I copied the dependencies from somewhere (probably either the example code of the atom notebook), and I don't actually know package.json syntax.

npm is pretty bandwidth-hungry, so I'd rather not be downloading alternative versions with my current (poor) internet connection.